roster/tests.py

```diff
@@ -1052,3 +1052,76 @@ def test_link_assistant(self) -> None:
         self.assertEqual(alice.assistant.pk, assistant.pk)
         resp = self.assertGetOK("link-assistant")
         self.assertEqual(len(resp.context["form"].fields["student"].queryset), 1)
+
+    def test_cancel_inquiry_sets_status_to_canceled(self):
+        alice = StudentFactory.create()
+        unit = UnitFactory.create()
+        inquiry = UnitInquiry.objects.create(
+            student=alice,
+            unit=unit,
+            action_type="INQ_ACT_UNLOCK",
+            status="INQ_NEW",
+            explanation="Please unlock",
+        )
+        self.login(alice)
+        resp = self.post(
```

Comment:
should check the status code is 200

Reply:
from `evan_django_tools` there's a shortcut for just `self.assertPost20X` replacing `self.post` for this